src/types.ts

```diff
@@ -209,6 +211,20 @@ export type TurnOffNamesAction = {
   };
 };
 
+export type TurnOnSeveralTranscriptsAction = {
+  type: OutgoingActionType.TURN_ON_SEVERAL_TRANSCRIPTS;
+  payload: {
+    track_ids: string[];
```

Comment:
Here and below — why this payload? You aren't using it when sending the message to genome browser.

Reply:
true. that was a copy paste mistake. Thanks for picking that.

Reply:
Nothing; from your updates to send.ts, the type should be sufficient.

Reply:
We reviewed my comment at a daily scrum; and I realised I was wrong. I misread track ids as transcript ids. Please ignore my comment.